app/src/main/java/org/mozilla/fenix/tabstray/browser/InactiveTabsAdapter.kt

```diff
@@ -107,7 +106,7 @@ class InactiveTabsAdapter(
         val items = tabs.list.map { Item.Tab(it) }
         val footer = Item.Footer(context.autoCloseInterval)
 
-        submitList(listOf(Item.Header) + items + listOf(Item.RecentlyClosed, footer))
```

Comment:
Should we also remove the [class declaration](https://github.com/mozilla-mobile/fenix/blob/16a3b92d342f9c5cbb3731ae6390fd44346923c7/app/src/main/java/org/mozilla/fenix/tabstray/browser/InactiveTabsAdapter.kt#L152) ? I'm not sure if we are using it anymore.

Reply:
I wasn't sure about this either. Are we removing it temporarily or are we not revisiting this again. 

I guess I can remove it all since we can always revisit it from the git history.

Reply:
Yeah, I'm not sure neither. I just saw we were only using it here 😄 . Either way for me it's Ok. Just wanted to confirm.

Reply:
👍🏽